lib/phoenix/live_dashboard/components/card_usage_component.ex

```diff
@@ -4,7 +4,7 @@ defmodule Phoenix.LiveDashboard.CardUsageComponent do
   def render(assigns) do
     ~L"""
     <div class="card progress-section mb-4">
-      <%= live_component @socket, Phoenix.LiveDashboard.TitleBarComponent, class: "card-body", percent: percentage(@usage, @limit) do %>
+      <%= live_component @socket, Phoenix.LiveDashboard.TitleBarComponent, id: "#{@id}-title-bar", class: "card-body", percent: percentage(@usage, @limit), csp_nonces: @csp_nonces do %>
```

Comment:
Unfortunately this makes the component stateful. We may want to call this "dom_id".

Reply:
@josevalim A lot of components already had an `id`. Should that one be removed then?

Reply:
Some of them may be required to be stateful. I know these ones do not though, so let's just make sure we are not adding those.